tests/caasp/stack_configure.pm

```diff
@@ -38,6 +38,15 @@ sub velum_config {
     mutex_create 'VELUM_CONFIGURED';
 }
 
+# Upload autoyast profile
+sub upload_autoyast {
+    send_key 'alt-tab';    # switch to xterm
+    assert_screen 'xterm';
+    assert_script_run 'curl --insecure --location --remote-name ' . get_var('DASHBOARD_URL') . '/autoyast';
+    upload_logs('./autoyast', failok => 1);
```

Comment:
please rename to "autoyast.xml" and remove "failok => 1" - if there is no profile it's a bug